### ISSUE TITLE
Oscillating validation loss

### ISSUE THREAD

**Issue by hamidrezafazlali** [tensorflow/tensorflow#23529]
I am training a deep CNN for image dehazing, and during training I plot the training and the validations losses. I plot the validation loss every 1 epoch. I do all the process on Tensorflow framework in Ubuntu 16 . The problem is my training loss decrease really well but my validation loss oscillates and it does not have a decreasing shape. Is it normal?

Here are the images of training and validation loss plots. The orange one is the training loss and the blue one is the validation loss.

The x-axis on the train plot is the iteration number and the x-axis on the validation plot is the epoch number.

![train](https://user-images.githubusercontent.com/32719272/48010891-7daa0c00-e0ec-11e8-95b3-863e66b0f044.png)

![validation](https://user-images.githubusercontent.com/32719272/48010905-84d11a00-e0ec-11e8-9502-bc0eb65fd177.png)

I really appreciate if you help me with this problem to figure out what part of my approach is wrong

**Comment by ymodak** [comment]
This question is better asked on [StackOverflow](http://stackoverflow.com/questions/tagged/tensorflow-tensorflow) since it is not a bug or feature request. There is also a larger community that reads questions there.

**Comment by hamidrezafazlali** [comment]
> This question is better asked on [StackOverflow](http://stackoverflow.com/questions/tagged/tensorflow-tensorflow) since it is not a bug or feature request. There is also a larger community that reads questions there.

I asked this question in stackoverflow and they told me to aski it here and it is not related to them!!!!
